.travis.yml

```diff
@@ -57,6 +59,10 @@ after_success:
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push $DOCKER_IMAGE:$DOCKER_TAG;
     fi
+  - if [ "$DOCKER_TAG" == "master" ]; then
```

Comment:
I agree. (I like the pr prefix name'ing)

What I'm saying is that every commit on master would be tagged (so there is no need for keeping it up to date) You are just pushing duplicate images with different tags.